### ISSUE TITLE
Same item can be scheduled twice on the same sitting

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tamzi/bungeni-portal#321]
```
What steps will reproduce the problem?
1.login as a clerk user
2.add a sitting / or browse to an existing sitting
3.drag and drop an item to schedule on the sitting
4.drag and drop the item again on the sitting

What is the expected output?
It should be allowed to schedule an item only once for a sitting

What do you see instead?
Item can be scheduled twice


```

Original issue reported on code.google.com by `ashok.ha...@gmail.com` on 30 Jul 2009 at 1:03
- Merged into: #420
